lambdas/pkgselect/index.py

```diff
@@ -194,9 +195,20 @@ def lambda_handler(request):
             key=key,
             sql_stmt=sql_stmt
         )
-        # Parse the response into a logical folder view
-        df = pd.read_json(result, lines=True)
-        response_data = file_list_to_folder(df)
+
+        if result is None:
+            response_data = file_list_to_folder(pd.DataFrame())
+        else:
+            # Parse the response into a logical folder view
+            df = pd.read_json(
+                result,
+                lines=True,
+                dtype=dict(
+                    logical_key='string',
+                    physical_key='string'
```

Comment:
In an actual manifest it's `physical_keys`.

Reply:
Isn't this column name in the result set?

Reply:
Yes, @sir-sigurd, the column name refers to the DataFrame (result set) and not the manifest.

Reply:
Find but why are we changing column names?